tests/test_job.py

```diff
@@ -943,7 +972,90 @@ def test_dependents_are_met_if_dependency_is_deleted(self):
 
         assert dependent_job.dependencies_are_met()
         assert dependent_job.get_status() == JobStatus.QUEUED
-    
+
+    def test_dependencies_are_met_at_execution_time(self):
+        queue = Queue(connection=self.testconn)
+
+        queue.enqueue_call(fixtures.say_hello, job_id="A")
+        queue.enqueue_call(fixtures.say_hello, job_id="B")
+        job_C = queue.enqueue_call(fixtures.check_dependencies_are_met, job_id="C", depends_on=["A", "B"])
+
+        w = Worker([queue])
+        w.work(burst=True)
+
+        assert job_C.result
+
+    def test_rpush_fixture(self):
+        fixtures.rpush('foo', 'bar')
+        assert self.testconn.lrange('foo', 0, 0)[0].decode() == 'bar'
+
+    def test_start_worker_fixture(self):
+        queue = Queue(name='testing', connection=self.testconn)
+        queue.enqueue(fixtures.say_hello)
+        conn_kwargs = self.testconn.connection_pool.connection_kwargs
+        fixtures.start_worker(queue.name, conn_kwargs, 'w1', True)
+        assert not queue.jobs
```

Comment:
Mind splitting these two tests to a separate `FixtureTestCase` class?

Reply:
I looked around, and most existing subclasses of `RQTestCase` seem to live in their own module called `test_something.py`, so I followed suit with a new `test_fixtures.py`. Not sure if that's what you had in mind.

Also followed suit with the `from __future__ import.....` line at the top of this module, which most of the `test_somethings.py` seem to include, though I'll admit I don't really know if that's needed or not. Just trying to keep it uniform.